### DISCUSSION TITLE
How to uninstall?

### DISCUSSION THREAD

**Discussion by beedell-roke** [oskarsh/Yin-Yang#268] [Q&A] [answered]
There are no instructions.

**Comment by l0drex** [comment, marked as answer]
Just run `scripts/uninstall.sh`

**Reply by beedell-roke** [reply]
> https://github.com/oskarsh/Yin-Yang/discussions/268#discussioncomment-8459615

@l0drex, ~~that doesn't appear to work.~~ Considering it requires interaction, you might want to inform the user of that if they run it via double-click (headlessly) from the file manager. Additionally, it didn't remove `$HOME/.local/share/applications/Yin-Yang.desktop` in my case. Regardless, thanks.
